ethereum/env/.env.template.mainnet

```
@@ -0,0 +1,45 @@
+# =============================================================================
+# RPC and Verification Configuration
+# =============================================================================
+
+RPC_URL=""
+
+# Custom forge arguments for contract deployment
+FORGE_ARGS=""
+
+# Verifier settings (contracts must be manually verified if omitted)
+# example VERIFY_ARGS="--verifier blockscout --verifier-url https://chain.blockscan.com/api"
+VERIFY_ARGS=""
+
+# =============================================================================
+# Wormhole Core Contract Migrations
+# =============================================================================
+
+# These values are unlikely to change unless announced
+INIT_SIGNERS=["0x58CC3AE5C097b213cE3c81979e1B9f9570746AA5"]
+INIT_GOV_CHAIN_ID=0x1
+INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
+
+# Wormhole chain ID
+INIT_CHAIN_ID=
+
+# EVM chain ID
+INIT_EVM_CHAIN_ID=
+
+# =============================================================================
+# Bridge Contract Migrations -- Only required when WTT/TokenBridge is deployed
+# =============================================================================
+
+# These values are unlikely to change unless announced
+BRIDGE_INIT_GOV_CHAIN_ID=0x1
+BRIDGE_INIT_GOV_CONTRACT=0x0000000000000000000000000000000000000000000000000000000000000004
```

Maybe we could reuse the definitions?
```suggestion
BRIDGE_INIT_GOV_CHAIN_ID=$INIT_GOV_CHAIN_ID
BRIDGE_INIT_GOV_CONTRACT=$INIT_GOV_CONTRACT
```

And similar things could be done for other parameters like the Wormhole chain id too.